### DISCUSSION TITLE
Help needed to understand the working of the application and some questions regarding GSoC'23.

### DISCUSSION THREAD

**Discussion by v1nshul** [koordinates/kart#759] [Google Soc / SoD]
Hello, I am a CS student from UK looking forward to send my application to Koordinates for GSoC'23. There are some questions I had regarding the organization and applications, it will be highly appreciated if these are answered. 

Questions for the working of application: 
1. I believe the application allows us to deal with tons of GeoSpatial data, I have tried to understand the fundamentals but I couldn't get a broad perspective on this software. It would be appreciated if there is somehow a simpler way of getting the idea. 
[sorry if the above question is not to the standards] 
2.  Can we perform operations on the data using QGIS ? 
3. Only these questions for now, but will have more in the future. 

Questions regarding GSoC'23: 
1.  Is the "Discussions" Section the only option we have to communicate with the mentors and contributors ?  If there is any other platform, then I would like to know which one is preferred by the mentors to communicate. 
2. How many contributors will be selected ? 

Thank You

**Comment by rcoup** [comment]
Hi, we haven't decided whether to participate in GSoC 2023 yet... (and if we do, whether we'll be accepted) — check back at the end of February

> I believe the application allows us to deal with tons of GeoSpatial data, I have tried to understand the fundamentals but I couldn't get a broad perspective on this software. It would be appreciated if there is somehow a simpler way of getting the idea.

There's a [brief introduction talk & demo](https://www.youtube.com/watch?v=fAIh6p4rczY) from FOSS4G UK 2022 as to what we're trying to do.

> Can we perform operations on the data using QGIS ?

There's a QGIS plugin available for Kart: see https://github.com/koordinates/kart-qgis-plugin
